devnet/node.yaml

```diff
@@ -93,34 +93,34 @@ spec:
             - ws://eth-devnet:8545
             - --neonRPC
             - ws://eth-devnet:8545
-            - --terraWS
-            - ws://terra-terrad:26657/websocket
-            - --terraLCD
-            - http://terra-terrad:1317
-            - --terraContract
-            - terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-            - --terra2WS
-            - ws://terra2-terrad:26657/websocket
-            - --terra2LCD
-            - http://terra2-terrad:1317
-            - --terra2Contract
-            - terra14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9ssrc8au
-            - --algorandAppID
-            - "4"
-            - --algorandIndexerRPC
-            - http://algorand:8980
-            - --algorandIndexerToken
-            - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-            - --algorandAlgodRPC
-            - http://algorand:4001
-            - --algorandAlgodToken
-            - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+            # - --terraWS
+            # - ws://terra-terrad:26657/websocket
+            # - --terraLCD
+            # - http://terra-terrad:1317
+            # - --terraContract
+            # - terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
+            # - --terra2WS
+            # - ws://terra2-terrad:26657/websocket
+            # - --terra2LCD
+            # - http://terra2-terrad:1317
+            # - --terra2Contract
+            # - terra14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9ssrc8au
+            # - --algorandAppID
+            # - "4"
+            # - --algorandIndexerRPC
+            # - http://algorand:8980
+            # - --algorandIndexerToken
+            # - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+            # - --algorandAlgodRPC
+            # - http://algorand:4001
+            # - --algorandAlgodToken
+            # - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
             - --solanaContract
             - Bridge1p5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o
-            - --solanaWS
-            - ws://solana-devnet:8900
-            - --solanaRPC
-            - http://solana-devnet:8899
+            # - --solanaWS
+            # - ws://solana-devnet:8900
+            # - --solanaRPC
+            # - http://solana-devnet:8899
```

Comment:
Are you commenting these so other people can see what they'll look like? I'm generally :-1: on updating code and committing it with a comment, but I'm guessing this is for users to be able to see how to run with everything?

Reply:
Was on the fence about this, because it does kinda act as a one stop shop for parameters and is really confusing to any sane person who wouldn't think to check the yaml _and_ the tiltfile